lib/web_ui/lib/src/engine/view_embedder/flutter_view_manager.dart

```diff
@@ -99,6 +99,11 @@ class FlutterViewManager {
     return _jsViewOptions[viewId];
   }
 
+  /// Returns the host element in which the Flutter view associated to [viewId] is embedded (if any).
+  DomElement? getHostElement(int viewId) {
```

Comment:
Do we have a clear definition of "host element"? If not, then the docs seem recursive. `getHostElement` already implied that it returns some "host element". There are two candidates for what people might think "host element" refers to:

- The `<flutter-view>` element.
- The direct parent of `<flutter-view>`.

The docs should explain which one this function returns.

Reply:
I'd expect the `hostElement` to be the **parent element of the flutter view**.

Reply:
That is what we have for `embeddingStrategy.hostElement`:
https://github.com/flutter/engine/blob/928d34f86a36df59985799e33c4f6dd2a6a5dfe6/lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/embedding_strategy.dart#L29-L30

Also in the [Flutter docs](https://docs.flutter.dev/platform-integration/web/initialization):
```
HTML Element into which Flutter renders the app. 
```

I think `rootElement` is used for the `<flutter-view>`.

Reply:
Doc in the code can be more specific though. How about updating doc for the `embeddingStrategy.hostElement` to be the following:

```dart
/// The dom element in which the Flutter view is embedded. 
/// This element is the direct parent element of the <flutter-view> element.
```

And then for the current function just leave it like this:
```dart
/// Returns the dom element in which the Flutter view associated to [viewId] is embedded (if any).
```

Reply:
@p-mazhnik is the "(if any)" true? I think there's always a "parent element of the flutter view" (unless viewId is wrong)

Reply:
Ah, I meant that there might not be a Flutter view associated with the provided `viewId`. I see now how this could be misinterpreted. Let's just omit it:
```
Returns the dom element in which the Flutter view associated to [viewId] is embedded
```

Reply:
The revised doc comment is clearer. Thank you! However, it is more important to get the docs right in `dart:ui_web`, which is part of the public API. I have some suggestions there.